decode/evm_rpc_test.go

```diff
@@ -17,13 +16,9 @@ var (
 	}()
 )
 
-func TestExtractBlockNumberFromEVMRPCRequestReturnsExpectedBlockForValidRequest(t *testing.T) {
+func TestUnitTestExtractBlockNumberFromEVMRPCRequestReturnsExpectedBlockForValidRequest(t *testing.T) {
 	requestedBlockNumberHexEncoding := "0x2"
-	requestBlockNumber, valid := cosmosmath.NewIntFromString(requestedBlockNumberHexEncoding)
-
-	if !valid {
-		t.Fatalf("failed to convert %s to cosmos sdk int", requestedBlockNumberHexEncoding)
-	}
```

Comment:
@galxy25, let me know if this deleted code was an important condition of this test.

the sdkmath parsing resulted in a test failure:
```
=== RUN   TestUnitTestExtractBlockNumberFromEVMRPCRequestReturnsExpectedBlockForValidRequest
    evm_rpc_test.go:39: 
                Error Trace:    /Users/pirtle/projects/kava/kava-proxy-service/decode/evm_rpc_test.go:39
                Error:          Not equal: 
                                expected: math.Int(math.Int{i:(*big.Int)(0xc00017d780)})
                                actual  : int64(2)
                Test:           TestUnitTestExtractBlockNumberFromEVMRPCRequestReturnsExpectedBlockForValidRequest
```

`NewIntFromString("0x2")` is 824635283328